src/Pages/DetailsPage.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import Details from '../Components/Details';
+import Footer from '../Components/Footer';
+import Nav from '../Components/Nav';
+import { fetchCoins } from '../redux/Coin/coinSlice';
+
+function DetailsPage() {
+  const dispatch = useDispatch();
+  const myCoins = useSelector((state) => state.coin.coins);
+  const { id } = useParams();
+  const coin = (myCoins.filter((coin) => coin.id === id))[0];
+
+  useEffect(() => {
+    let active = true;
+    (async () => {
+      if (active) {
+        dispatch(fetchCoins());
+      }
+    })();
+    return () => {
+      active = false;
+    };
+  }, [dispatch]);
+
+  return (
+    <div className="app_container">
```

Comment:
- [ ] Good job 👏  But I can not find the back button that takes us to home 😢  According to the project requirement, your app should have a Home page and Detail pages and the user should be able to navigate the detail page by clicking the individual category item and the user should move from the `Detail` page to the `Home` page by clicking the "Back" button. Please make sure to implement this to make your project complete and work as expected :+1: